Makefile

```diff
@@ -2,21 +2,61 @@
 build-and-push-daemon:
 	bash scripts/build-and-push-daemon.sh
 
+# dev-testnet
+.PHONY: build-daemon-dev-testnet
+build-daemon-dev-testnet:
+	bash scripts/build-daemon.sh dev-testnet
+
+.PHONY: push-daemon-dev-testnet
+push-daemon-dev-testnet:
+	bash scripts/push-daemon.sh
+
 .PHONY: deploy-lambdas-dev-testnet
 deploy-lambdas-dev-testnet:
-	yarn workspace wallet-service run serverless deploy --stage dev-testnet --region eu-central-1
+	AWS_SDK_LOAD_CONFIG=1 yarn workspace wallet-service run serverless deploy --stage dev-testnet --region eu-central-1
+
+# testnet
+.PHONY: build-daemon-testnet
+build-daemon-testnet:
+	bash scripts/build-daemon.sh testnet
+
+.PHONY: push-daemon-testnet
+push-daemon-testnet:
+	bash scripts/push-daemon.sh testnet
 
 .PHONY: deploy-lambdas-testnet
 deploy-lambdas-testnet:
-	yarn workspace wallet-service run serverless deploy --stage testnet --region eu-central-1
+	AWS_SDK_LOAD_CONFIG=1 yarn workspace wallet-service run serverless deploy --stage testnet --region eu-central-1
+
+# mainnet-staging
+
+.PHONY: build-daemon-mainnet-staging
+build-daemon-mainnet-staging:
+	bash scripts/build-daemon.sh mainnet_staging
```

Comment:
Do we still need to pass this argument to the script?

Also, if not, do we still need a build and a push commands in the Makefile for each environment?